src/Pages/PricingPage.js

```diff
@@ -0,0 +1,339 @@
+import React, { Component } from "react";
+import { css } from "glamor";
+import { observer } from "mobx-react";
+import { withRouter } from "react-router-dom";
+
+import PricingTierCard from "./../Components/PricingTierCard";
+
+const TimeText = ({ topText, bottomText }) => (
+  <div
+    className={css({
+      display: "flex",
+      flexDirection: "column",
+      alignItems: "center"
+    })}
+  >
+    <p
+      className={css({
+        fontSize: "46px"
+      })}
+    >
+      {topText}
+    </p>
+    <p
+      className={css({
+        fontSize: "14px"
+      })}
+    >
+      {bottomText}
+    </p>
+  </div>
+);
+
+const FaqQna = ({ question, answer }) => (
+  <div
+    className={css({
+      marginTop: "45px"
+    })}
+  >
+    <p
+      className={css({
+        fontSize: "22px",
+        fontWeight: "600"
+      })}
+    >
+      {question}
+    </p>
+    <p
+      className={css({
+        fontSize: "22px",
+        fontWeight: "300",
+        marginTop: "25px"
+      })}
+    >
+      {answer}
+    </p>
+  </div>
+);
+
+const PricingPageObserver = observer(
+  class PricingPage extends Component {
+    componentWillReact() {
+      console.log("I will re-render, since the observables changed!");
+    }
+
+    render() {
+      let faqs = [];
+
+      for (let i = 0; i < 9; i++) {
```

Comment:
Haven't seen one of these out in the wild for a while! Since it is building mock data it is probably fine :)